_data/simple-icons.json

```diff
@@ -2053,6 +2053,14 @@
             "source": "https://beatstars.world/brand-guidelines",
             "guidelines": "https://beatstars.world/brand-guidelines"
         },
+        {
+            "title": "Beekeeper Studio",
+            "hex": "FAD83B",
+            "source": "https://www.beekeeperstudio.io",
```

Comment:
We should use their [press kit](https://www.beekeeperstudio.io/press/) as source here.